src/Metadata/Extractor/XmlResourceExtractor.php

```diff
@@ -450,4 +453,23 @@ private function buildGraphQlOperations(\SimpleXMLElement $resource, array $root
 
         return $data;
     }
+
+    private function buildPersistenceMeans(\SimpleXMLElement $resource): ?PersistenceMeansInterface
+    {
+        $persistenceMeans = $resource->persistenceMeans ?? null;
+        if (!isset($persistenceMeans)) {
+            return null;
+        }
+        if (1 !== \count($persistenceMeans)) {
+            throw new \DomainException('There can only be one configured persistence means. "resources.xsd" needs to be fixed.');
+        }
+        $elasticsearchDocument = $persistenceMeans->elasticsearchDocument ?? null;
+        if (isset($elasticsearchDocument)) {
```

Comment:
```suggestion
        if ($elasticsearchDocument ?? false) {
```

Reply:
Use `??` instead of `isset()` everywhere it's possible.

Reply:
Is this for performance reasons?

Reply:
`isset` was in fact not needed (see https://github.com/api-platform/core/pull/5297#discussion_r1060466524)